scripts/add_crashid_to_queue.py

```diff
@@ -0,0 +1,13 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The problem with the shebang thing is that it requires you to be in the right environment such that the correct python is used. The original script this is based on had `#!/data/socorro-virtualenv/bin/python` or some such nonsense which doesn't work in the docker environment.

I left off the shebang in an attempt to make it easier to run these scripts in both the docker and admin node environments.

That problem goes away once we switch to the new docker infrastructure. So this is a "what do we do for now" kind of thing.

I'll re-add the shebang line and make it docker-environment-friendly.